airbyte-webapp-e2e-tests/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
-  "include": ["./**/*.ts"],
+  "include": ["./**/*.ts", "cypress/plugins/index.js"],
```

Comment:
Should we maybe shortly conver the `plugins/index.js` as well into a TS file (since it's anyway more or less content free), so we won't need this special addition here? (not a bloking concern for me)